From: =?utf-8?q?Piotr_O=C5=BCarowski?= <piotr@debian.org>
Date: Tue, 20 Jun 2017 20:13:42 +0200
Subject: disable privacy breach links in documentation

---
 docs/conf.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 86a3961..dc23b53 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -165,12 +165,12 @@ html_theme = "alabaster"
 html_theme_options = {
     "logo": "yarl-icon-128x128.png",
     "description": "Yet another URL library",
-    "github_user": "aio-libs",
-    "github_repo": "yarl",
-    "github_button": True,
-    "github_type": "star",
-    "github_banner": True,
-    "codecov_button": True,
+    # "github_user": "aio-libs",
+    # "github_repo": "yarl",
+    # "github_button": True,
+    # "github_type": "star",
+    # "github_banner": True,
+    # "codecov_button": True,
     "pre_bg": "#FFF6E5",
     "note_bg": "#E5ECD1",
     "note_border": "#BFCF8C",
