From ca968581e8dcc074b56c50693f0890082d230d4c Mon Sep 17 00:00:00 2001
From: Sam Bull <aa6bs0@sambull.org>
Date: Mon, 9 Jan 2023 23:10:14 +0000
Subject: [PATCH] Fix 3.11 tests (#810)

---
 tests/test_url_parsing.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/test_url_parsing.py b/tests/test_url_parsing.py
index 0a7f5f1d..0f0dd7ee 100644
--- a/tests/test_url_parsing.py
+++ b/tests/test_url_parsing.py
@@ -73,8 +73,8 @@ def test_not_a_scheme1(self):
         assert u.fragment == ""
 
     def test_not_a_scheme2(self):
-        u = URL("37signals:book")
-        assert u.scheme == "37signals"
+        u = URL("signals37:book")
+        assert u.scheme == "signals37"
         assert u.host is None
         assert u.path == "book"
         assert u.query_string == ""
